canonical-web-and-design/ubuntu.com
# Update the /managed page

## Done
- Updated the copy on the /managed page.

## QA
- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/managed
- Check the content matches the [copy doc](https://docs.google.com/document/d/17zJ4vTutqVpn0XzUI784BHdJov8cGW1ttDwIjsvcEzE/edit#heading=h.qp7hfvggs6wp)

## Issue / Card
Fixes https://github.com/canonical-web-and-design/web-squad/issues/3074

## Screenshots
![image](https://user-images.githubusercontent.com/1413534/92255351-19b02480-eeca-11ea-9c23-4a429a5064f2.png)
